### DISCUSSION TITLE
White flash/fade when navigating between pages in admin

### DISCUSSION THREAD

**Discussion by JoeDotCom** [JulianPrieber/llc-docker#79] [Issues]
Hiya.
Is there a way to remove the "fade" effect from the admin UI page loads? I work in dark mode and when navigating in the linkstack UI, I clock a link (example: /studio/add-link) the page abruptly loads a 100% white background and then slowly fades to the dark theme. This results in a very bright screen and then my eyes needing to reacclimatize to the dark theme. 
any way to stay in dark mode without the white "changing pages" flashes?
Cool docker image. I hope i can work this out. :)

**Comment by JulianPrieber** [comment]
Documentation:
https://templates.iqonic.design/hope-ui/documentation/html/dist/main/loadingTransitions.html
